### ISSUE TITLE
Extend Extents.jl predicates?

### ISSUE THREAD

**Issue by rafaqz** [JuliaGeo/GeometryOps.jl#242]
This could reduce ecosystem name clashes

**Comment by asinghvi17** [comment]
That makes sense, let's make sure they are all the same though!

**Comment by asinghvi17** [comment]
Actually nevermind...we'd have to pirate these lines

https://github.com/rafaqz/Extents.jl/blob/ced3c5fa4c1761e6138e661a74acc6c8a72c5f9f/src/Extents.jl#L142-L146

and we can't depend on GeoInterface in Extents, because it's circular...

**Comment by asinghvi17** [comment]
So maybe we need a GeometricPredicateFunctionStubs.jl package that just has these function stubs, and then Extents can override them for its own use.

**Comment by rafaqz** [comment]
Yeah but maybe we should remove those methods anyway as only GI.extent reliably works on geometries. It all needs a rethink at some point
